frontend/src/screens/GameScreen/gameScreen.tsx

```diff
@@ -0,0 +1,247 @@
+    const [englishWords, setEnglishWords] = useState<string[]>()
+    const [turkishWords, setTurkishWords] = useState<MaybeWordPair[]>()
+    const [submitted, setSubmitted] = useState(false);
+    const navigation = useNavigation()
+    const { sessionId } = props.route.params;
+    const [currentRoundId, setCurrentRoundId] = useState<UID>("");
+    const [isLoading, setIsLoading] = useState(false);
+
+    useEffect(() => {
+        console.log("sessionId = " + sessionId)
+        if (currentRoundId == "") {
+            fi.startRound(sessionId).then(result => setCurrentRoundId(result));
+        } else {
+            fi.getUser().then(user => {
+                fi.getRoundPairs(currentRoundId).then(words => {
+                    setEnglishWords(durstenfeldShuffle(props.shuffleFunction(words)));
+                    setTurkishWords(durstenfeldShuffle(words.map((word, i) => ({ turkish: word.turkish, correctEnglishWord: word.english, english: undefined }))));
+                    setSubmitted(false)
+                    setIsLoading(false)
+                }).catch(console.error);
+            }).catch(console.error);
+        }
+    }, [currentRoundId]);
+
+    if (englishWords === undefined) {
+        return <LoadingScreen />
+    }
+
+    const restartRound = () => {
+        fi.endRound(currentRoundId);
+        fi.startRound(sessionId).then(result => setCurrentRoundId(result));
+    }
+
+    const renderEnglishOptions = () => {
+        return englishWords?.map(word => {
+            if (turkishWords?.some(({ english }) => english === word) ?? false) {
+                return (<View key={word} style={styles.englishUsed} />)
+            }
+            return (<DraxView payload={word} key={word}
+                style={styles.draxView}
+                draggingStyle={{ opacity: 0.3 }}
+                dragReleasedStyle={{ opacity: 0.3 }}
+                longPressDelay={100}>
+                <Text style={styles.english}>{word}</Text>
+            </DraxView>)
+        })
+    }
+
+    const wordDroppedFunction = (turkishWords: MaybeWordPair[], word: MaybeWordPair, i : number) => {
+      return (newWord : string) => {
+      const newTurkishWords = turkishWords.map(({ english, turkish, correctEnglishWord }) => {
+            if (english === newWord) {
+                return { turkish, correctEnglishWord }
+            } else {
+                return { turkish, correctEnglishWord, english }
+            }
+        })
+        newTurkishWords[i] = { english: newWord, turkish: word.turkish, correctEnglishWord: word.correctEnglishWord }
+        setTurkishWords(newTurkishWords)
+      }
+    }
+
+    const removeWordFunction = (turkishWords: MaybeWordPair[], word: MaybeWordPair, i: number) => {
+      return () => {
+        const newTurkishWords = [...turkishWords]
+        newTurkishWords[i] = { turkish: word.turkish, correctEnglishWord: word.correctEnglishWord }
+        setTurkishWords(newTurkishWords)
+    }
```

Comment:
This lambda is missing an indentation - though this is not on you and more of a sign that I should add prettier to dev